README.md

```diff
@@ -76,3 +102,33 @@ Hydra is already optimised for adding, updating and removing pages, staff, advic
 
 * Exposed as a data file to give clients better access.
 * Set in the *Data* / *Footer* section.
+
+
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/hello. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
```

Comment:
The GH address needs to be https://github.com/CloudCannon/hydra-jekyll-template/pulls

Reply:
Can we remove the http://contributor-covenant.org stuff?